scss/shared.scss

```diff
@@ -32,7 +32,6 @@
 }
 
 .custom-select {
-  border-radius: 0;
```

Comment:
This is covered by the [mixin](https://github.com/mozilla/network-pulse/pull/1102/files#diff-5a69ecdcb6159ed08581c1ee5d81d240R47).

Reply:
If we want to have one mixin for both search "input" and "dropdown" I would get rid of this `custom-dropdown` class and move the rules to the mixin to keep things all in one place and avoid rule duplicates.

Reply:
ahh, good point 👍